docs/src/site/apt/index.apt.vm

```diff
@@ -70,14 +70,26 @@ mvn -Preporting site
 * Site Publication
 
     This POM prepares site publication to {{{/developers/website/index.html}Apache Maven's site svnpubsub}}.
-    Every inheriting POM needs to define <<<maven.site.path>>> property
-    with relative path to <<<$\{project.artifactId}-LATEST>>> publication uri, and define <<<distributionManagement>>> to avoid
-    automatic inheritance from parent:
+
+    There are defined <<<maven.site.path>>> properties in parent POM for each supported component type, as:
 
 +-----+
   <properties>
     <maven.site.path>xxx-archives/\${project.artifactId}-LATEST</maven.site.path>
   </properties>
++-----+
+
+    For components extensions, plugins and shared which target to Maven 4.x we have properties in parent POM:
```

Comment:
I'm not sure what "shared" is here. Is a noun missing?
also just target, not target to

Reply:
I thought here about project modules which have defined such property